htmlToElement.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
-import {StyleSheet, Text} from 'react-native';
+import {StyleSheet, Text, Dimensions, View} from 'react-native';
```

Comment:
`View` is unused (except in comments). Use of `Dimensions` is interesting, but outside the scope of your PR.

Reply:
ops, sorry! i made a mess on this PR. I was testing something and shared my master (with a lot of tests and bad code just to see how it works.) Soon I will send the proper fixes. Ignore this PR complete. And thank you for your review.

Reply:
@flaviohenriquecbc That's ok, thanks for thinking of us! Very happy to take a look at your next offering :smile: